### ISSUE TITLE
Convert remaining onscreen displays to QML

### ISSUE THREAD

**Issue by torriem** [torriem/QtAgOpenGPS#12]
Convert to QML:
Reverse Indicator
Lift Indicator
Lost RTK
RTK Age

**Comment by torriem** [comment]
Also No GPS screen

**Comment by Davidwedel** [comment]
And add the settings for compass/speedo

**Comment by Davidwedel** [comment]
Svenn arrow also?

**Comment by torriem** [comment]
No, only the 2D elements like the light bar, compass, speedo, Simulator On, etc.  Things that are fixed to the display on top of everything else.  

Anything that's drawn on the field using field coordinates is still going to be in OpenGL

**Comment by Davidwedel** [comment]
Compass moved from ogl to qml.https://github.com/Davidwedel/QtAgOpenGPS/commit/4df83d4d7758f56e7c9dcba46787f43793f51794
I think RTK age and lost rtk also https://github.com/Davidwedel/QtAgOpenGPS/commit/05c92bf666b12c223ae54fa5b09041cb8bca8177

**Comment by Davidwedel** [comment]
hyd lift moved over

**Comment by Davidwedel** [comment]
No GPS, and reverse arrow moved to qml.
